test/fixtures/output/login/1.golden

```diff
@@ -1,4 +1,4 @@
 - is_current: true
-  name: login-on-prem@example.com-http://127.0.0.1:\d+\?cacertpath=.+test[/\\]fixtures[/\\]input[/\\]login[/\\]test\.crt
+  name: login-on-prem@example.com-http://127.0.0.1:\d+\?cacertpath=.*test[/\\]fixtures[/\\]input[/\\]login[/\\]test\.crt
```

Comment:
It's not a required flag.

Rather, this is because I accidentally thought that the line which returns the absolute path was specific to the legacy context, so I removed it.

The result is that the test only returned the relative path, which is why it failed with `.+`. I'll revert this change and add back the code that makes the path absolute.